UnityProject/Assets/Scripts/Systems/Research/Objects/ResearchServer.cs

```diff
@@ -10,8 +10,18 @@
 
 namespace Systems.Research.Objects
 {
-	public class ResearchServer : NetworkBehaviour, IMultitoolMasterable
+	public class ResearchServer : NetworkBehaviour, IMultitoolMasterable, IServerSpawn, IServerDespawn
 	{
+		public int RP {
```

Comment:
```suggestion
		public int RP => techweb.researchPoints ?? 0;
```

Reply:
![Capture](https://user-images.githubusercontent.com/48405920/202891509-9a4407eb-2cdd-4512-8cef-a385c6755b85.PNG)

Reply:
Since techweb isn't a unity object then you can just use `techweb?.researchPoints ?? 0;` instead.

Reply:
Cool thanks, commited

Reply:
My bad, I thought it was a unity object